config/vufind/searchspecs.yaml

```diff
@@ -70,6 +70,63 @@
 #
 # ...etc.
 #
+# GlobalExtraParams allows extra parameters to be (conditionally) added to the Solr
+# search request:
+#
+# GlobalExtraParams:
+#   - param: bf
+#     value: "recip(rord(publishDateSort),1,1000,1000)"
+#     conditions:
+#       - NoDismaxParams
+#         - bf
+#         - bq
+#       - SearchTypeIn:
+#         - AllFields
+#         - Title
+#         - Author
+#       - SearchTypeNotIn:
+#         - Series
+#
+# Value can be a single string or a list of strings to repeat the parameter.
+#
+# Available conditions:
+#
+# - NoDismaxParams: None of the search types specifies any of the listed parameters
+#   in DismaxParams
+# - SearchTypeIn      At least one of the search types must be in the given list
+# - AllSearchTypesIn  All of the search types must be in the given list
+# - SearchTypeNotIn   None of the search types must be in the given list
+#
+# Note that conditions are processed in the order of occurrence. SearchTypeNotIn is
```

Comment:
This note is no longer true and should be removed, correct?

Reply:
Yes, removed.